### PR TITLE
Update docs for react SDK

### PR DESCRIPTION
**Pull request by antho1404**
This PR aims to include the core library in the documentation.
To do that, I used jsdoc to document all the code directly in the codebase, ensuring nice autocompletion and supporting the documentation website.
The react library has also been updated with new parameters.
The example app has been updated (and greatly simplified into a single page) accordingly to the new js library and uses a function to explore the core package possibilities.
I had to reorganize the documentation with a new SDK directory that contains core & react.
I also updated the home page that was already obsolete and missing webhook & services.
Finally, I removed all unnecessary mdx files.

### CODE REVIEW

File: docs/pages/SDK/react/useCreateNFT.md [hunk 1]
```diff
@@ -19,23 +18,28 @@ export default function Component() {
...
-      traits: [
-        { type: 'Head', value: 'Cap' },
-        { type: 'Body', value: 'Samurai' },
-      ], // Array of traits associated to the NFT
-    })
+    const isLazyMinted = true
+    await createNFT(
+      {
+        chain: 1, // chainId of the network to mint on
+        collection: '0x0000', // address of the collection to use
```
Comment (ismailToyran) [inline docs/pages/SDK/react/useCreateNFT.md]: Can we also add `supply` and `royalties` here as an example even though they are not required.

File: docs/pages/SDK/react/useCreateOffer.md [hunk 2]
```diff
@@ -9,28 +9,25 @@ Hook for creating an offer, accessing the active process step as well as the tra
...
-      currencyId: '56-0x19a4866a85c652eb4a2ed44c42e4cb2863a62d51', // Id of the currency in Liteflow API
-      auctionId: '06511214-38d8-11ed-a261-0242ac120002', // Optional, id of the auction to create the offer on.
-      expiredAt: '06511214-38d8-11ed-a261-0242ac120002'
-        ? null
-        : new Date('2025-12-17T03:24:00'), // Optional date at which the offer expired. If not set, the API will enforce one. If offer is on an auction, this date is always set by the API.
-      takerAddress: '0xaA6510ac710e97D2A6a280658B5f641dA79891FE', // Optional, wallet address of the offer's receiver. This allows to create an offer that only a specific wallet can accept.
+      unitPrice: {
+        amount: BigNumber.from(1).mul(BigNumber.from(10).pow(18)), // this value is in base unit of the token. This is equivalent to 1 ETH. Use BigNumber
+        currencyId: '0x19a4866a85c652eb4a2ed44c42e4cb2863a62d51', // Currency address
+      },
```
Comment (ismailToyran) [inline docs/pages/SDK/react/useCreateOffer.md]: Same comment for `taker`, `expiredAt`, `auctionId`

File: docs/pages/_meta.json [hunk 3]
```diff
@@ -3,6 +3,6 @@
   "graphql-api": "GraphQL API",
   "webhook": "Webhook",
   "service": "Service",
-  "hooks": "React Hooks",
```
Comment (ismailToyran) [inline docs/pages/_meta.json:6]: Strange behaviour on the left column.
Need to either change folder name to be `sdk` from `SDK` (I prefer this one)
Or need to change line 6 on this file to;
`"SDK": "SDK",`

File: packages/react/README.md [hunk 4]
```diff
@@ -1,60 +1,71 @@
...
-      type: OfferType.Buy,
-      quantity: BigNumber.from(1),
-      unitPrice: BigNumber.from(1).mul(BigNumber.from(10).pow(18)), // this value is in base unit of the token. This is equivalent to 1 ETH. Use BigNumber
-      assetId: 'XXX',
-      currencyId: 'XXX',
+      type: 'BUY',
+      chain: 1, // Ethereum
+      collection: `0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d`,
+      token: '100',
+      quantity: {
```
Comment (ismailToyran) [inline packages/react/README.md]:
```suggestion
      unitPrice: {
```

File: docs/pages/SDK/react/index.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  const { data: signer } = useSigner()
+  const [createOffer] = useCreateOffer(signer)
+
+  const handleClick = async () => {
+    const offerId = await createOffer({
+      type: 'BUY',
+      chain: 1, // Ethereum
+      collection: `0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d`,
+      token: '100',
+      quantity: {
```
Comment (ismailToyran) [inline docs/pages/SDK/react/index.md]:
```suggestion
      unitPrice: {
```

**Review by ismailToyran — APPROVED** [review]
Some fixes and some suggestions,
Didn't find any other issue. Looks amazing with the new core + react documentation 🎉

File: docs/pages/SDK/react/useCreateNFT.md (hunk 1 above)
Comment (antho1404) [reply docs/pages/SDK/react/useCreateNFT.md]: done

File: docs/pages/SDK/react/useCreateOffer.md (hunk 2 above)
Comment (antho1404) [reply docs/pages/SDK/react/useCreateOffer.md]: done, didn't add the auctionId tho as it is a different type of offer

File: packages/react/README.md (hunk 4 above)
Comment (antho1404) [reply packages/react/README.md]: nice catch

File: docs/pages/SDK/react/index.md (hunk 5 above)
Comment (antho1404) [reply docs/pages/SDK/react/index.md]: nice catch

File: docs/pages/_meta.json (hunk 3 above)
Comment (antho1404) [reply docs/pages/_meta.json:6]: this has been fixed, it was an issue with the case that git didn't automatically moved from initially SDK to sdk

**Pull request merged**